projects/plugins/jetpack/modules/contact-form/admin.php

```diff
@@ -465,6 +465,46 @@ function grunion_manage_post_columns( $col, $post_id ) { // phpcs:ignore Variabl
 	}
 }
 
+add_action( 'restrict_manage_posts', 'grunion_source_filter' );
+/**
+ * Add a post filter dropdown at the top of the admin page.
+ *
+ * @return void
+ */
+function grunion_source_filter() {
+	$screen = get_current_screen();
+
+	if ( 'edit-feedback' !== $screen->id ) {
+		return;
+	}
+
+	$parent_id = intval( isset( $_GET['jetpack_form_parent_id'] ) ? $_GET['jetpack_form_parent_id'] : 0 ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
What is the reason why the nonce sniff is being ignored here? It looks like a returned GET value is being used in `form_posts_dropdown`, so it looks like something is being changed in response to the request? (referencing this post: p9dueE-4z8-p2 )

Reply:
It's a `GET` request and so it's not supposed to be changing anything, the value is being used just to make sure the dropdown's default options matches the currently applied filter. So I guess it falls under 'not taking any action' - should I add a relevant comment in there?

Reply:
Thanks for explaining! Yes the additional comment is a good idea here (like on line 890 of admin.php) is a good idea.